src/server/models/cacheable_queries/organization-contact.js

```diff
@@ -24,7 +24,7 @@ const organizationContactCache = {
       })
       .first();
 
-    if (r.redis) {
+    if (r.redis && organizationContact) {
```

Comment:
Need to check if organizationContact exists before trying to save it to redis, or else the code will run into an error